src/guides/v2.2/cloud/architecture/scaled-architecture.md

```diff
@@ -0,0 +1,36 @@
+---
+group: cloud-guide
+title: Scaled architecture
+functional_areas:
+  - Cloud
+---
+
+The Cloud infrastructure scales according to your resource needs to achieve greater efficiency. The {{site.data.var.ece}} monitors your applications and automatically adjusts capacity to maintain steady, predictable performance. Converting to this architecture helps to mitigate problems, such as latency, large spikes in traffic, or running out of storage.
+
+## Split architecture
+
+Historically, the Pro architecture consisted of 3 nodes, each containing a full tech stack. Now, the scalable infrastructure provides a tiered solution with a minimum of 6 nodes: 3 nodes for the web server and 3 nodes for the database and other services. This split architecture provides the capability to scale tiers independently to achieve an optimal balance of performance. The database tier scales vertically (increases in size), and the web tier scales horizontally (increases instance count) and vertically (changes instance type and size).
+
+### Database tier scaling
+
+There are 3 nodes for the database and services. When the database (db) tier approaches capacity, the only way to scale is to increase the server size, such as boosting the CPU power and memory. Capacity is limited to the size of the node that is available. Because the database cluster is designed for high availability, you cannot scale horizontally in a reliable way with the technologies used.
+
+![Database tier scaling]
+
+Further optimizing performance includes routing traffic. You can route traffic based on the node type. For example, you can serve PHP traffic on the db node or you can isolate the database node from the PHP traffic. Although, db nodes do not serve PHP traffic by default.
```

Comment:
`serve PHP traffic`
It will be better to rename PHP traffic to WEB traffic.
Because it can be confusing.
PHP traffic for me, for example, is traffic from PHP script like queries to DB, Redis, etc. Core nodes serve this traffic. But by default core nodes do not serve WEB traffic.

Reply:
Well, if it is in the section regarding database and services, then it would be core traffic? If it is better to use Web as an example, then perhaps this is the wrong place for this comment?